Tasks/Common/clienttool-common/ClientToolUtilities.ts

```diff
@@ -0,0 +1,200 @@
+import AdmZip = require('adm-zip');
+import * as vsts from 'azure-devops-node-api';
+import os = require("os");
+import * as path from "path";
+import * as tl from "azure-pipelines-task-lib";
+import * as toollib from "azure-pipelines-tool-lib/tool";
+import { IRequestOptions } from 'azure-devops-node-api/interfaces/common/VsoBaseInterfaces';
+
+
+export function getClientToolLocation(dirName: string, toolName: string): string {
+    let toolPath: string = path.join(dirName, toolName);
+    if (tl.osType() !== "Windows_NT") {
```

Comment:
I'm wondering if there was supposed to be something else here. Otherwise, this check doesn't do anything different than line 11 that I can see